course/models/models.py

```diff
@@ -300,13 +300,16 @@ def get_lines(self):
             ]
             random.shuffle(lines)
             rendered_lines.append({
-                'name': input_files['name'],
+                'name': render_text(input_files['name'], self.get_variables()),
```

Comment:
Have tried submitting a code solution with this?
I think we should do a little more than this. Probably we have to change some stuff if the graders.

Reply:
I have to double-check... I think I tested and it worked but I'll make sure.